### PR TITLE
i18n(ja): Add new `cloudflare.mdx`  translation

### PR DESCRIPTION
**Pull request by piro-hiroki**
#### Description (required)

Translate `cloudflare.mdx` into Japanese！

file: [guides/deploy/cloudflare.mdx](https://github.com/withastro/docs/blob/main/src/content/docs/en/guides/deploy/cloudflare.mdx)

#### Related issues & labels (optional)

- Closes #
- Suggested label:

### CODE REVIEW

**Comment by sarah11918** [comment]
Hi @piro-hiroki ! Just checking if this PR should still be marked as a draft?

If it's ready for editing, then please change the status so that it is ready for review! We are hoping to merge as many PRs by next week as we can before the docs site code migrates to Starlight. So if this is ready, we would like to be able to review it.  Thanks!

**Comment by sarah11918** [comment]
Hi @piro-hiroki ! Is this one ready for a review now?

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:115]:
```suggestion
    モードが設定されていない場合、デフォルトは`"advanced"`です。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:117]:
```suggestion
2. **ディレクトリ**モード： 関数を`directory`モードで実行したい場合にこのモードを使用します。つまり、アダプタはアプリのクライアントサイド部分を同じようにコンパイルしますが、ワーカースクリプトをプロジェクトルートの`functions`フォルダに移動します。アダプタはそのフォルダに`[[path]].js`を置くだけなので、プラグインやページのミドルウェアを追加して、バージョン管理でチェックすることができます。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:124]:
```suggestion
### Pages Functionsの使用
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:126]:
```suggestion
[Pages Functions](https://developers.cloudflare.com/pages/platform/functions/)により、専用のサーバーを実行することなしに、サーバーサイドのコードを実行して動的な機能を実現できます。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:130]:
```suggestion
📚 [AstroでのSSR](/ja/guides/server-side-rendering/)についてもっと読む。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:134]:
```suggestion
エラーが発生する場合は、ローカルで使用している`node`のバージョン（`node -v`）が環境変数で指定しているバージョンと一致しているかどうかを再確認してください。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:136]:
```suggestion
Cloudflareは[Node v16.13](https://miniflare.dev/get-started/cli#installation)を必要とします。このバージョンはAstroが必要とする最小値よりも新しいので、少なくともv16.13を使用していることを再確認してください。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:138]:
```suggestion
CloudflareのAuto Minify設定の結果、クライアントサイドのハイドレーションが失敗することがあります。コンソールに`Hydration completed but contains mismatches`と表示されたら、Cloudflare settingsのAuto Minifyを無効にしてください。
```

File: src/content/docs/ja/guides/deploy/cloudflare.mdx
Comment (morinokami) [inline src/content/docs/ja/guides/deploy/cloudflare.mdx:63]: https://github.com/withastro/docs/pull/5284

```suggestion
pnpm add wrangler --save-dev
```

**Review by morinokami — CHANGES_REQUESTED** [review]
@sarah11918 Added some suggestions! I think we can merge this PR once those suggestions are merged.

@piro-hiroki Please let me know if you have anything to add!

**Review by sarah11918 — APPROVED** [review]
:rocket:

**Pull request merged**
